src/index.js

```diff
@@ -28,14 +28,17 @@ export const getSortedObjectPaths = (obj) => {
  *  `babel-plugin-transfor-define` take options of two types: static config and a path to a file that
  *  can define config in any way a user sees fit. getReplacements takes the options and will either
  *  return the static config or get the dynamic config from disk
- * @param  {Object|String}  configOptions  configuration to parse
```

Comment:
We probably need to update the documentation above this method correctly describe the method after the changes.

If we update `getReplacements` to perform like I outlined above, it could possibly something like the following.

```js
/**
 * `babel-plugin-transfor-define` takes options in the form of a static config. If a filePath attribute
 * exists on the configOptions object, getReplacements will get the dynamic config from disk. The file
 * can define config in any way a user sees fit. getReplacements will return configurations that is the
 * static config object with any filePath config spread onto it.
 * @param  {Object} configOptions configuration to parse
 * @param  {string} configOptions.filePath path to dynamic configuration file
 * @return {Object} replacement object
 */
```